openmicroscopy/openmicroscopy
# Jstree followup

This PR adds version number to jstree lib directory and allows user to Drag and Drop Dataset & Screen onto experimenter to make then orphaned.

To test:
 - Drag a Dataset from under a Project onto Experimenter.
 - Should become orphaned
 - Same for a Plate under a Screen
 - Cannot drag orphaned Dataset or Screen and drop onto Experimenter
 - Quick check that jstree still works on Tags and Public pages (no missing library, css, images etc from re-naming).
